app/src/main/java/ru/sonya/week3/model/CatsDAO.kt

```diff
@@ -0,0 +1,24 @@
+package ru.sonya.week3.model
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.Query
+
+
+@Dao
+interface CatsDAO {
+
+    @Insert
+    fun insert(vararg cats: RoomCat)
+
+    @Delete
+    fun delete(cat: RoomCat)
+
+    @Query("SELECT * FROM DBCats")
+    fun getAllCats(): List<RoomCat>?
```

Comment:
```suggestion
    fun getAllCats(): List<RoomCat>
```
Если будет пусто, то вернёт пустой список